client/actions/editActions.js

```diff
@@ -30,6 +35,14 @@ export const changeView = (activeView) => ({
   }
 })
 
+export const viewArticles = emitAction(() => ({
+  type: actions.VIEW_ARTICLES,
+  key: messageTypes.articlesRequested,
+  payload: {
+    timestamp: new Date().toISOString()
+  }
+}))
+
```

Comment:
If this is for viewing the article list it might make sense to create a new reducer for articles.

Reply:
absolutely, I'm still working on getting things working, but I'll start reorganizing/moveing some code units afterwards

Reply:
Awesome, looking forward to seeing where this goes

Reply:
👍 I'll let you know when it's in a good spot for an initial review 🙂